### ISSUE TITLE
Incorrect Resource ID Format for Importing eventhub_namespace_authorization_rule in Documentation

### ISSUE THREAD

**Issue by rudolphjacksonm** [terraform-providers/terraform-provider-azurerm#6879]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
```
terraform: 0.12.24
azurerm: 2.6.0
```

### Affected Resource(s)

* `azurerm_eventhub_namespace_authorization_rule`

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior

Importing a eventhub_namespace_authorization_rule resource as per the example shown in the docs should work.

### Actual Behavior

Received an error about a missing parameter, which oddly doesn't match the fix I've raised in the PR mentioned below:

```
Error: Error making Read request on Azure EventHub Authorization Rule : eventhub.NamespacesClient#GetAuthorizationRule: Invalid input: autorest/validation: validation failed: parameter=authorizationRuleName constraint=MinLength value="" details: value length must be greater than or equal to 1
```

### Steps to Reproduce

1. `terraform import azurerm_eventhub_namespace_authorization_rule /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example/providers/Microsoft.EventHub/namespaces/namespace1/authorizationRules/rule1`

Now, try again but make the 'A' in `authorizationRules` uppercase. This will work:
`terraform import azurerm_eventhub_namespace_authorization_rule /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example/providers/Microsoft.EventHub/namespaces/namespace1/AuthorizationRules/rule1

### References

* #6878

**Comment by neil-yechenwei** [comment]
Thanks for opening this issue. I've made a [minor change](https://github.com/terraform-providers/terraform-provider-azurerm/pull/6886) on it if you don't mind.

**Comment by tombuildsstuff** [comment]
hey @rudolphjacksonm 

Thanks for opening this issue.

As you've mentioned the fix for this has been merged in #6878 - this'll get published onto the Terraform Website when we release version 2.10 of the Azure Provider later this week.

Thanks!

**Comment by rudolphjacksonm** [comment]
Awesome!!! Thanks for the quick response on this guys!
